### PR TITLE
Auto unparsing based on definition, limit coverage to semantic rules

### PR DESCRIPTION
**Pull request by ehildenb**
Fixes: #41

-   Update KWasm submodule (BLOCKED on kframework/wasm-semantics#227)
-   `translateCoverage.py` only converts semantic rules, not function rules.
-   Using auto-unparser which comes via `pyk. instead of manually written one.

### CODE REVIEW

**Comment by virgil-serbanuta** [comment]
Looks good to me, but I don't know enough about this to be sure.

**Comment by virgil-serbanuta** [comment]
BTW, why are those json files single-line files? It makes it fairly hard to figure out what the changes are (I gave up at some point).

**Comment by ehildenb** [comment]
@virgil-serbanuta the JSON files are very large, storing them as multi-line would take up quite a bit more disk space. These files are auto-generated, but the generation process takes about 20min and 30GB of RAM. So I just commit them to the repo, and use the committed versions instead of re-generating them when needed. So no need in general to review the JSON files.

File: Jenkinsfile [hunk 1]
```diff
@@ -25,29 +25,21 @@ pipeline {
             '''
           }
         }
-        stage('Polkadot') {
-          steps {
-            sh '''
-              make deps-polkadot
-            '''
-          }
-        }
```
Comment (hjorthjort) [inline Jenkinsfile]: Don't we still want to test it? Or is it just way too expensive to run?

File: pykWasm.py [hunk 2]
```diff
@@ -8,184 +8,49 @@
...
-
-for symb in WASM_TEXT_underbar_unparsed_symbols:
-    WASM_symbols[symb] = underbarUnparsingInModule('WASM-TEXT', symb)
-
-ALL_symbols = pyk.combineDicts(pyk.K_symbols, WASM_symbols)
+WASM_definition_llvm_no_coverage_dir = '.build/defn/kwasm/llvm'
+WASM_definition_llvm_coverage_dir    = '.build/defn/coverage/llvm'
+
+WASM_definition_haskell_no_coverage_dir = '.build/defn/kwasm/haskell'
+WASM_definition_haskell_coverage_dir    = '.build/defn/coverage/haskell'
```
[the remark this answers is not in the document]
Comment (ehildenb) [reply pykWasm.py]: This just loads the json definitions into memory, but doesn't deal with the coverage reports themselves (which is what line 135 of the Makefile is dealing with).

File: translateCoverage.py [hunk 3]
```diff
@@ -29,13 +36,28 @@ def _fatal(msg, exit_code = 1):
 with open(src_rules_file, 'r') as src_rules:
     src_rule_list = [ rule_hash.strip() for rule_hash in src_rules ]
```
[the remark this answers is not in the document]
Comment (ehildenb) [reply translateCoverage.py]: Whole file has been commented now.

File: Jenkinsfile (hunk 1 above)
Comment (ehildenb) [reply Jenkinsfile]: I added the tests back in.

**Pull request merged**